docs/docs/tutorial.md

```diff
@@ -39,9 +39,9 @@ For this tutorial, we'll use prebuilt JavaScript files on a CDN. Open up your fa
 <html>
   <head>
     <title>Hello React</title>
-    <script src="https://fb.me/react-{{site.react_version}}.js"></script>
-    <script src="https://fb.me/JSXTransformer-{{site.react_version}}.js"></script>
-    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.0/react.js"></script>
```

Comment:
We should leave this with `{{site.react_version}}` in there. That ensures we don't need to update this page for every release.